PowerEditor/src/WinControls/WindowsDlg/WindowsDlg.cpp

```diff
@@ -534,18 +535,29 @@ BOOL WindowsDlg::onInitDialog()
 	lvColumn.mask = LVCF_WIDTH | LVCF_TEXT | LVCF_SUBITEM | LVCF_FMT;
 	lvColumn.fmt = LVCFMT_LEFT;
 
-	lvColumn.pszText = TEXT("\u21F5 Name");

```

Comment:
NON TRIVIAL CHANGE - `pszText` requires non-const strings so I used `CString` for that. It's a perfect candidate (better than `std::string`), since its `GetBuffer()` returns a non-const string pointer. Not possible with `std::string` until C++17 or so.